cores/esp32/USBCDC.h

```diff
@@ -33,6 +33,7 @@ typedef enum {
     ARDUINO_USB_CDC_LINE_CODING_EVENT,
     ARDUINO_USB_CDC_RX_EVENT,
     ARDUINO_USB_CDC_TX_EVENT,
+    ARDUINO_USB_CDC_OVERFLOW,
```

Comment:
let's name this `ARDUINO_USB_CDC_RX_OVERFLOW` for clarity (and the option to add TX in the future)

Reply:
done.